zulip_bots/zulip_bots/bots/flock/flock.py

```diff
@@ -62,7 +62,7 @@ def get_flock_response(content: str, config: Dict[str, str]) -> str:
         return error
 
     if len(str(recipient_id)) > 30:
-        return recipient_id
+        return "Found user is invalid."
```

Comment:
We should change the commit message here to reflect that you fixed the code itself.  I think `flock bot: Fix error message for invalid user.` would work.

Reply:
@showell I updated commit message. Also I replaced `local variable` with `local var` in commit messages.